tests/Client/Cursor.test.ts

```diff
@@ -40,38 +40,38 @@ describe('Cursor', () => {
     });
 
     it('advancing with empty or malformed array returns self', () => {
-      expect(start.advance([])).to.equal(start);
-      expect(start.advance([{}])).to.equal(start);
+      expect(start.advance([] as any)).to.equal(start);
+      expect(start.advance([{}] as any)).to.equal(start);
     });
 
     it('advancing with non-array returns self', () => {
-      expect(start.advance(undefined)).to.equal(start);
-      expect(start.advance(null)).to.equal(start);
-      expect(start.advance(new Error())).to.equal(start);
+      expect(start.advance(undefined as any)).to.equal(start);
+      expect(start.advance(null as any)).to.equal(start);
+      expect(start.advance(new Error() as any)).to.equal(start);
     });
   });
 
   describe('#toQuery()', () => {
     it('includes channel', () => {
-      expect(new Cursor('channel').toQuery()).to.eql({channel: 'channel'});
+      expect(new Cursor('channel').toQuery()).to.eql({ channel: 'channel' });
     });
 
     it('includes after if it is non-null', () => {
-      expect(new Cursor('channel', {after: 40}).toQuery()).to.eql({
+      expect(new Cursor('channel', { after: 40 as any }).toQuery()).to.eql({
```

Comment:
same, `after` should be defined as a number (`as any` is never the right solution)